js/widgets.js

```diff
@@ -1,108 +1,172 @@
-Array.range = function(n) {
-    // Array.range(5) --> [0,1,2,3,4]
-    return Array.apply(null, Array(n)).map((x, i) => i);
-};
+/* This file is using http://standardjs.com/ */
+// `fetch` polyfill added by webpack.
+/* global fetch */
 
-Object.defineProperty(Array.prototype, 'chunk', {
-    // Chunking an Array given the numeric parameter.
-    value: function(n) {
-        return Array.range(Math.ceil(this.length / n)).map((x, i) => this.slice( i * n, i * n + n));
-    }
-});
+// Constants in use within the HTML.
+const TERRITORY = 'data-udata-territory'
```

Comment:
Maybe be use underscores (`data-udata_territory`) as dashes are not accepted in the specification. `el.dataset['udata-territory'] = 'something'` will raise an exception.